src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SNI/SNIHandle.cs

```diff
@@ -11,6 +12,11 @@ namespace Microsoft.Data.SqlClient.SNI
     /// </summary>
     internal abstract class SNIHandle
     {
+        /// <summary>
+        /// TLS 1.3 (not fully supported) in addition to obsolete and insecure protocols are excluded.
+        /// </summary>
+        protected readonly SslProtocols SupportedProtocols = SslProtocols.Tls12 | SslProtocols.Tls11 | SslProtocols.Tls;
```

Comment:
Is this a breaking change?

Reply:
Since managed SNI is newer than the native SNI, I tried to remove the obsolete protocols here. The main purpose of this PR was just to exclude Tls 1.3; So, we decided to avoid a breaking change right now and include the obsolete SSL protocols.